### PR TITLE
Remove Planner from Model and Storage

### PR DESCRIPTION
**Pull request by OneArmyj**
With this PR:
- files will be saved and read from 4 independent files, accommodation, activity, contact, itinerary
- 12 files (11 tests file & PlannerBuilder.java) have been commented out 

Things that needs to be done:
- create TypicalAccommodation, AccommodationBuilder, TypicalActivity, ActivityBuilder, TypicalContact, ContactBuilder, TypicalItinerary, ItineraryBuilder
- integrate into commented tests files

### CODE REVIEW

File: src/main/java/seedu/address/logic/LogicManager.java [hunk 1]
```diff
@@ -29,24 +33,27 @@
 
     private final Model model;
     private final Storage storage;
-    private final PlannerParser plannerParser;
+    private final PlannerParser addressParser;
```
Comment (1nefootstep) [inline src/main/java/seedu/address/logic/LogicManager.java]: why is this called addressParser?

File: src/main/java/seedu/address/model/Itinerary.java [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+import seedu.address.model.day.Day;
+import seedu.address.model.day.UniqueDayList;
+import seedu.address.model.field.Name;
+
+
+/**
+ * Wraps all data at the address-book level
+ * Duplicates are not allowed (by .isSameContact comparison)
+ */
+public class Itinerary implements ReadOnlyItinerary {
```
Comment (1nefootstep) [inline src/main/java/seedu/address/model/Itinerary.java]: We don't have to specifically call it Itinerary anymore; we can just follow the convention and call it DayManager.

File: src/main/java/seedu/address/model/Itinerary.java [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+    private final UniqueDayList days;
```
Comment (1nefootstep) [inline src/main/java/seedu/address/model/Itinerary.java]: We need to consider if every day has to be "unique". (empty days are all the same?)

File: src/main/java/seedu/address/logic/LogicManager.java (hunk 1 above)
Comment (OneArmyj) [reply src/main/java/seedu/address/logic/LogicManager.java]: thanks for spotting, accidentally replace when i did a replace all in the file

File: src/main/java/seedu/address/logic/LogicManager.java [hunk 4]
```diff
@@ -29,24 +33,27 @@
...
 
     public LogicManager(Model model, Storage storage) {
         this.model = model;
         this.storage = storage;
-        plannerParser = new PlannerParser();
+        addressParser = new PlannerParser();
```
Comment (oscarsu97) [inline src/main/java/seedu/address/logic/LogicManager.java]: wrong naming here as well(:

Comment (oscarsu97) [inline src/main/java/seedu/address/logic/LogicManager.java]: wrong naming here as well(:

File: src/main/java/seedu/address/logic/LogicManager.java [hunk 5]
```diff
@@ -29,24 +33,27 @@
...
     }
 
     @Override
     public CommandResult execute(String commandText) throws CommandException, ParseException {
         logger.info("----------------[USER COMMAND][" + commandText + "]");
 
         CommandResult commandResult;
-        Command command = plannerParser.parseCommand(commandText);
+        Command command = addressParser.parseCommand(commandText);
```
Comment (oscarsu97) [inline src/main/java/seedu/address/logic/LogicManager.java]: wrong naming here as well(:

File: src/main/java/seedu/address/logic/LogicManager.java (hunk 4 above)
Comment (OneArmyj) [reply src/main/java/seedu/address/logic/LogicManager.java]: changed, thanks

File: src/main/java/seedu/address/logic/LogicManager.java (hunk 5 above)
Comment (OneArmyj) [reply src/main/java/seedu/address/logic/LogicManager.java]: changed, thanks

**Pull request merged**